src/udiskslinuxencrypted.c

```diff
@@ -408,8 +459,24 @@ handle_unlock (UDisksEncrypted        *encrypted,
   /* calculate the name to use */
   if (is_in_crypttab && crypttab_name != NULL)
     name = g_strdup (crypttab_name);
+  else {
+    if (is_luks)
+      name = g_strdup_printf ("luks-%s", udisks_block_get_id_uuid (block));
+    else
+      /* TCRYPT devices don't have a UUID, so we use the device number instead */
+      name = g_strdup_printf ("tcrypt-%lu", udisks_block_get_device_number (block));
```

Comment:
Please the `G_GUINT64_FORMAT` macro here, `%lu` doesn't work on 32 bit machines.

Reply:
Done in 77818e04.